blockchain/genesis/genesis.go

```diff
@@ -6,9 +6,103 @@
 
 package genesis
 
-const (
-	// BlockGasLimit is the total gas limit could be consumed in a block
-	BlockGasLimit = uint64(20000000)
-	// ActionGasLimit is the per action gas limit cap
-	ActionGasLimit = BlockGasLimit / 4
+import (
+	"flag"
+
+	"github.com/pkg/errors"
+	"go.uber.org/config"
+	"go.uber.org/zap"
+
+	"github.com/iotexproject/iotex-core/address"
+	"github.com/iotexproject/iotex-core/pkg/keypair"
+	"github.com/iotexproject/iotex-core/pkg/log"
+)
+
+// DefaultAdminPrivateKey is used to create the default genesis config. It could facilitate quick setup of the
+// blockchain, but it MUST NOT be used in production.
+const DefaultAdminPrivateKey = "bace9b2435db45b119e1570b4ea9c57993b2311e0c408d743d87cd22838ae892"
```

Comment:
This is part of the config, not part of the genesis, in addition, how could all BPs bootstrap with this secret value?

Reply:
1. This is part of genesis, and it **MUST** be the same for all BPs. Like mentioned in the comment above, `DefaultAdminPrivateKey` is introduced to make default genesis self-contained. By having this value, we can easily set default for those protocols need somewhat initial address/owner/admin/... (e.g., rewarding protocol's initial admin) If we want to start a blockchain to play with, we could quickly start a default genesis. In real production, this *MUST NOT** be used. For example, rewarding protocol's initial admin will be overwritten with an address that only its real owner has the associated private key.

2. All BPs have their own different secrets, which is via config now. That's still there, and different from this value.